x/monitoringp/keeper/begin_block_test.go

```diff
@@ -3,64 +3,61 @@ package keeper_test
 import (
 	"testing"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/stretchr/testify/require"
 	abci "github.com/tendermint/tendermint/abci/types"
 
-	spntypes "github.com/tendermint/spn/pkg/types"
+	tc "github.com/tendermint/spn/testutil/constructor"
 	testkeeper "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/testutil/sample"
 	"github.com/tendermint/spn/x/monitoringp/types"
 )
 
 func TestKeeper_ReportBlockSignatures(t *testing.T) {
-	k, _, ctx := testkeeper.MonitoringpKeeper(t)
-
-	// simplified type for abci.VoteInfo for testing purpose
-	type vote struct {
-		address string
-		signed  bool
-	}
-
-	// create LastCommitInfo from list of addresses and signed boolean
-	lastCommit := func(votes ...vote) abci.LastCommitInfo {
-		var lci abci.LastCommitInfo
-
-		// add votes
-		for _, vote := range votes {
-			lci.Votes = append(lci.Votes, abci.VoteInfo{
-				Validator: abci.Validator{
-					Address: []byte(vote.address),
-				},
-				SignedLastBlock: vote.signed,
-			})
-		}
-
-		return lci
+	// valSet is a validator set for testing purpose that represent consensus addresses
+	// association with operator addresses
+	type val struct {
+		opAddr   string
+		consAddr []byte
 	}
+	type valSet []val
```

Comment:
Type is unused.  Will this be used by a later PR?

Reply:
That some dust from a previous implementation thanks for spotting